Remove Active Development CLI Download

BrandonJenkins14 opened the pull request:
Active Development CLI package on Zowe.org is outdated and should be removed. We can reintroduce download buttons when more forward features are added to @latest CLI or another Zowe component.

zowe/docs-site#1140 is open to temporarily remove the Active Development documentation. 

@MarkAckert or ZLC, can you ensure that the changes here don't disrupt anything besides the CLI download? I moved the disclaimer statement about Zowe Binaries up near the primary download buttons.

BrandonJenkins14 commented:
I think I answered my own question - this PR throws off the formatting of the sections and nests them under pre-release builds. I will see if I can resolve.

MarkAckert commented:
> Not sure what's wrong; the section tags are still in place. Could use assistance.

I'll take a look.

BrandonJenkins14 commented:
I fixed the layout in last commit. Removed the "Past Releases" expandable section. Also removed an announcement about Active Dev version. 

**Note:**
- there is an announcement about SMP/E that might be outdated. 
- there is an active-dev config file in `_data` , should we remove it?

MarkAckert commented on _config.yml:
```diff
@@ -21,7 +21,6 @@ conformance_page_url: https://www.openmainframeproject.org/projects/zowe/conform
 zos_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?version=
 cli_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?type=cli&version=
 cli_plugins_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?type=cli-plugins&version=
-cli_active_development_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?type=cli-active-development&version=
```
Please leave this field in the _config.yml, but with empty value:
```
cli_plugins_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?type=cli-plugins&version=	
cli_active_development_download_url:
smpe_download_url: https://d1xozlojgf8voe.cloudfront.net/legal.html?type=smpe&version=	
```

MarkAckert commented on index.md:
```diff
@@ -126,26 +127,15 @@ The easiest way to get started with Zowe is by downloading the convenience build
   </ul>
 </p>
 </details>
-{% if site.cli_active_development_download_url %}
```
Please leave this block in the index.md, in case we re-use it in the future. With the blank value for cli_active_development_download_url, it won't appear.

BrandonJenkins14 commented:
LGTM?

BrandonJenkins14 commented:
Can we merge this? Not sure if any users have clicked on this button lately, but if they do they will definitely be confused by this outdated version.

BrandonJenkins14 commented:
It seems that good things come to those who wait :P Someone else removed Active Development download from the live site in a separate PR :) 

Closing this PR.
